src/test/java/integration/ShipmentControllerIT.java

```diff
@@ -95,7 +95,7 @@ public void getShipment_notFound() throws Exception {
     @Test
     @SuppressWarnings("unchecked")
     public void createShipment() throws Exception {
-        Client sender = testHelper.createClientAsSender();
+        Client sender = testHelper.createClientAndCounterparty();
```

Comment:
Creating client by default means that Counterparty should be created, so no need to mention this in the method name.
Plz, rename to createClient